crates/devices/virtio-blk/src/stdio_executor.rs

```diff
@@ -356,7 +356,7 @@ impl<B: Backend> StdIoBackend<B> {
                         .map_err(Error::Write)?;
                 }
             }
-            RequestType::Flush => return self.inner.fsync().map(|_| 0).map_err(Error::Flush),
+            RequestType::Flush => return self.inner_mut().fsync().map(|_| 0).map_err(Error::Flush),
```

Comment:
Why do you use the function call and not the field as it was before?

Reply:
I mistakenly modified this one (and a few others) as if they were a tests. I've reverted those changes.